### ISSUE TITLE
Unofficial plugin indicator

### ISSUE THREAD

**Issue by dsisco11** [dsisco11/SR_Plugin_Loader#18]
Add an indicator to the plugin settings window that tells users when a certain plugin is not listed in the MASTER_PLUGINS_LIST.json file.
This will make it harder for developers to forget to add their plugins.

**Comment by dylanrenwick** [comment]
Would it perhaps be beneficial to, rather than simply hosting a json file in git for this, have a web interface where developers can submit their plugins, and then simply have an auto-generated json file for approved plugins?

**Comment by dsisco11** [comment]
yes that would be ideal but creating a whole website for that is a little out of scope at the moment. We could make the loader check multiple sources though.
But downloading a huge list of JSON info about plugins each time the loader starts can really get inefficient. So the way it works atm is it caches the master list locally and checks its hash against the one on github, only redownloading when it changes. So ideally any alternate lists it checks should provide a way to check the list itself for changes.

**Comment by dylanrenwick** [comment]
I have a domain, a web server, php skills, and a lot of free time, I could look into setting up a website for this kind if thing if you'd like. It should be fairly simple to offer a hash of the json file for change checks.

**Comment by dsisco11** [comment]
I mean if you wanna do that then feel free, we can easily add in a second plugin list source.
